### PR TITLE
add letsencrypt to hub

### PR DESCRIPTION
**Pull request by thetechnocrat-dev**
Should I move this into it's own separate playbook so it does't run on every deploy?

Screenshot 2023-05-31 at 5 14 06 PM

### CODE REVIEW

**Comment by hevans66** [comment]
I recommend moving this to `provision_jupyter.yaml`. I see it as a thing that gets run when you are setting up jupyter.

File: infrastructure/ansible/jupyter_deploy_plex.yaml [hunk 1]
```diff
@@ -60,6 +60,15 @@
         dest: /opt/tljh/config/jupyterhub_config.d/environmnet.py
         mode: "755"
 
+    - name: Enable HTTPS via letsencrypt
+      become: yes
+      ansible.builtin.shell:
+        cmd: |
+          tljh-config set https.enabled true
+          tljh-config set https.letsencrypt.email josh@labdao.xyz
```
Comment (hevans66) [inline infrastructure/ansible/jupyter_deploy_plex.yaml:68]: If we're being pedantic, I recommend moving the email to a variable, probably at the play level level for now. Will make it easier for someone to overwrite. If they provisioning for a different org.

Comment (thetechnocrat-dev) [reply infrastructure/ansible/jupyter_deploy_plex.yaml]: Good points, just implemented.

**Review by hevans66 — APPROVED** [review]
💯

**Pull request merged**